proxy/pkg/zdmproxy/clusterconn.go

```diff
@@ -149,13 +148,10 @@ func NewClusterConnector(
 	lastHeartbeatTime := &atomic.Value{}
 	lastHeartbeatTime.Store(time.Now())
 
-	streamIdMetrics, err := GetStreamIdsMetricsByClusterConnector(proxyMetrics, connectorType)
+	streamIdMetrics, err := GetNodeMetricsByClusterConnector(nodeMetrics, connectorType)
```

Comment:
you don't need this call any more right? you can just use the `nodeMetrics` parameter

Reply:
`nodeMetrics` is the parent struct, it contains `OriginMetrics`, `TargetMetrics` and `AsyncMetrics`. The helper function, already used in other places, will return the proper inner struct depending on the connector type.